it/src/main/java/org/corfudb/universe/scenario/fixture/Fixtures.java

```diff
@@ -57,149 +60,133 @@ private TestFixtureConst() {
+                    .dockerImage(CorfuServerParams.DOCKER_IMAGE_NAME);
+
+            universe = VmUniverseParams.builder()
+                    .vSphereUrl(vmProperties.getProperty("vsphere.url"))
+                    .networkName(vmProperties.getProperty("vm.network"))
+                    .templateVMName("corfu-server")
+
+                    .vmUserName(credentials.getProperty("vm.username"))
+                    .vmPassword(credentials.getProperty("vm.password"))
+                    .vSphereUsername(credentials.getProperty("vsphere.username"))
+                    .vSpherePassword(credentials.getProperty("vsphere.password"))
 
-        static ImmutableList<CorfuServerParams> buildMultipleServers(int numNodes, String clusterName) {
-
-            List<CorfuServerParams> serversParams = IntStream
-                    .rangeClosed(1, numNodes)
-                    .map(i -> ServerUtil.getRandomOpenPort())
-                    .boxed()
-                    .sorted()
-                    .map(port -> CorfuServerParams.serverParamsBuilder()
-                            .port(port)
-                            .clusterName(clusterName)
-                            .build()
-                    )
-                    .collect(Collectors.toList());
-
-            return ImmutableList.copyOf(serversParams);
+                    .cleanUpEnabled(true);
         }
 
-        static ImmutableList<CorfuServerParams> buildVmMultipleServers(int numNodes, String clusterName, String vmNamePrefix) {
-            List<CorfuServerParams> serversParams = new ArrayList<>();
+        @Override
+        public VmUniverseParams data() {
+            if (data.isPresent()) {
+                return data.get();
+            }
 
-            for (int i = 0; i < numNodes; i++) {
-                final int port = ServerUtil.getRandomOpenPort();
+            CorfuClusterParams clusterParams = cluster.build();
 
-                VmCorfuServerParams serverParam = VmCorfuServerParams.builder()
-                        .clusterName(clusterName)
-                        .vmName(vmNamePrefix + (i + 1))
-                        .mode(CorfuServer.Mode.CLUSTER)
-                        .logLevel(Level.TRACE)
-                        .persistence(CorfuServer.Persistence.DISK)
-                        .port(port)
-                        .stopTimeout(Duration.ofSeconds(1))
-                        .build();
+            FixtureUtil fixtureUtil = fixtureUtilBuilder.build();
+            ImmutableList<CorfuServerParams> serversParams = fixtureUtil.buildVmServers(
+                    clusterParams, servers, vmPrefix
+            );
+            serversParams.forEach(clusterParams::add);
 
-                serversParams.add(serverParam);
+            ConcurrentMap<String, String> vmIpAddresses = new ConcurrentHashMap<>();
+            for (int i = 0; i < clusterParams.getNumNodes(); i++) {
+                vmIpAddresses.put(vmPrefix + (i + 1), "0.0.0.0");
```

Comment:
Please put this into a ANY_ADDRESS constant.

Reply:
done